modules/raft/src/test/java/org/apache/ignite/internal/raft/RaftGroupServiceTest.java

```diff
@@ -617,7 +618,7 @@ private void mockUserInput(boolean delay, @Nullable Peer peer) {
                 argThat(new ArgumentMatcher<ActionRequest>() {
                     @Override
                     public boolean matches(ActionRequest arg) {
-                        return arg.command() instanceof TestWriteCommand;
+                        return arg instanceof WriteActionRequest && ((WriteActionRequest) arg).command() instanceof TestWriteCommand;
```

Comment:
It would be more convenient with a common interface that returns a command.

Reply:
Next issue will change the type of `command()` property for write commands into a byte[], interfaces will become incompatible

Reply:
Ok